client-spark/spark-3/src/main/scala/org/apache/spark/sql/execution/columnar/RssColumnBuilder.scala

```diff
@@ -28,7 +28,10 @@ trait RssColumnBuilder {
   /**
    * Initializes with an approximate lower bound on the expected number of elements in this column.
    */
-  def initialize(rowCnt: Int, columnName: String = "", useCompression: Boolean = false): Unit
+  def initialize(
+      rowCnt: Int,
+      columnName: String = "",
+      dictionaryEnabled: Boolean = false): Unit
```

Comment:
In order to adapt to new encoding types added later, this suggestion is changed to encodingEnabled